core/src/main/java/com/sequenceiq/cloudbreak/util/StackUtil.java

```diff
@@ -119,7 +137,13 @@
     }
 
     public Set<Node> collectReachableNodes(Stack stack) {
-        return hostOrchestrator.getResponsiveNodes(collectNodes(stack), gatewayConfigService.getPrimaryGatewayConfig(stack));
+        return hostOrchestrator.getResponsiveNodes(collectNodes(stack), gatewayConfigService.getPrimaryGatewayConfig(stack)).getReachableNodes();
+    }
+
+    public NodeResult collectReachableAndUnreachableNodes(Stack stack) {
+        NodeResult nodeResult = hostOrchestrator.getResponsiveNodes(collectNodes(stack), gatewayConfigService.getPrimaryGatewayConfig(stack));
+        LOGGER.debug("Node collection result: {}", nodeResult);
```

Comment:
logging out all of nodes with all of their parameters (based on toString of Node class) might be too large and would increase size of our log, I would log only some identifier of nodes or only hostnames or only just few parameters of the Node class

Reply:
Good catch, I will fix this.